ci/tests.yaml

```diff
@@ -19,5 +19,6 @@ dependencies:
   - scipy
   - setuptools_scm
   - pip:
-    - pyradiosky
+    - git+git://github.com/RadioAstronomySoftwareGroup/pyradiosky
```

Comment:
These changes are not compatible with the current release, because they rely on SkyModel.stokes array being of Quantity type. I figure we'd wait to merge this until a new release of pyradiosky, and then switch it back to using the released version before merging.

Reply:
Oh okay that makes a lot of sense. Let's make an issue so we don't forget to go back to the released version when able.

Reply:
We cut a new pyradiosky version (v0.1.0) last week and there are no new commits after that label, so I think you should be able to use the PyPI package.